packages/fxa-settings/src/pages/Index/en.ftl

```diff
@@ -0,0 +1,13 @@
+# Index / home page
+
+index-header = Enter your email
+index-sync-header = Continue to your { -product-mozilla-account }
+index-sync-subheader = Sync your passwords, tabs, and bookmarks everywhere you use { -brand-firefox }.
+# $serviceName - the name of the service which the user authenticating for
+index-subheader-with-servicename = Continue to { $serviceName }
+index-subheader-with-logo = Continue to <span>{ $serviceLogo }</span>
+index-subheader-default = Continue to account settings
+index-cta = Sign in or sign up
```

Comment:
The backbone version has "Sign up or sign in" (reversed)

Reply:
Thank you for catching! \o/

I'll make a note about the banner/error states in the functional ticket.